src/plugins/ConsolidatePlugin.ts

```diff
@@ -59,7 +57,10 @@ export class ConsolidatePluginClass implements Plugin {
     try {
       file.contents = await consolidate[this.engine].render(file.contents, {
         cache: false,
-        filename: 'base'
+        filename: 'base',; basedir;
+        basedir: (options.basedir !== undefined) ? options.basedir : file.context.homeDir;
+        includeDir: (options.includeDir !== undefined) ? options.includeDir : file.context.homeDir;
```

Comment:
```options``` should be ```this```

```basedir: (this.basedir !== undefined) ? this.basedir : file.context.homeDir;```

same for include

Reply:
Sorry I'm tired I didn't sleep I code wrong..